### ISSUE TITLE
Query failure

### ISSUE THREAD

**Issue by billmclaughlin** [datasert/realfire#22]
```
select DAY_IN_YEAR(CreatedDate), Count(Id)
from WhizzAssistChat__c 
where event__c='CHAT_ON_CLOSE'
group by DAY_IN_YEAR(CreatedDate)

```
"Error parsing [select DAY_IN_YEAR(CreatedDate), Count(Id)
from WhizzAssistChat__c 
where event__c='CHAT_ON_CLOSE'
group by DAY_IN_YEAR(CreatedDate)]. Error: missing EOF at '('"

works in Dev Console query runner just fine.

**Comment by brsanthu** [comment]
Hi Bill,

we confirm that this is a query parser issue. Will work on the issue and update you when we have a fix.

Santhosh

**Comment by datasert** [comment]
We confirm that this is an issue with Realfire query parser. Will try to fix the issue and update here.

**Comment by brsanthu** [comment]
Issue is fixed in Realfire 5.0.0
